shell/composables/useLabeledFormElement.ts

```diff
@@ -70,7 +70,9 @@ export const labeledFormElementProps = {
   }
 };
 
-export const useLabeledFormElement = (props: LabeledFormElementProps, emit: (event: string, ...args: any[]) => void): UseLabeledFormElement => {
+export type emitMethodType = (event: string, ...args: any[]) => void;
```

Comment:
Explained above, I needed to be able to type case the emit method elsewhere so I extracted the definition.